Public/Sdk/Public/Managed/Testing/XUnit/xunit.dsc

```diff
@@ -38,28 +38,28 @@ export function runConsoleTest(args: TestRunArguments): Result {
     const testClass  = args.className || Environment.getStringValue("[UnitTest]Filter.testClass");
 
     let arguments : Argument[] = CreateCommandLineArgument(testDeployment.primaryFile, args, testClass, testMethod);
-    
+
     let execArguments : Transformer.ExecuteArguments = {
         tool: args.tool || tool,
         tags: args.tags,
         arguments: arguments,
         dependencies: [
             testDeployment.contents,
         ],
-        warningRegex: "^(?=a)b", // This is a never matching warning regex. StartOfLine followed by the next char must be 'a' (look ahead), and the next char must be a 'b'. 
+        warningRegex: "^(?=a)b", // This is a never matching warning regex. StartOfLine followed by the next char must be 'a' (look ahead), and the next char must be a 'b'.
         workingDirectory: testDeployment.contents.root,
         retryExitCodes: Environment.getFlag("RetryXunitTests") ? [1] : [],
         unsafe: args.untrackTestDirectory ? {untrackedScopes: [testDeployment.contents.root]} : undefined,
     };
 
-    if (qualifier.targetFramework === "netcoreapp2.2") {
-        execArguments = importFrom("Sdk.Managed.Frameworks.NetCoreApp2.2").wrapInDotNetExeForCurrentOs(execArguments);
+    if (qualifier.targetFramework === "netcoreapp3.0") {
+        execArguments = importFrom("Sdk.Managed.Frameworks.NetCoreApp3").wrapInDotNetExeForCurrentOs(execArguments);

```

Comment:
again to keep parity for when 3.1 will come out, why not have the .0 already to match the TFM?

Reply:
Adjusted, see above.